pkgs/build-support/rust/sysroot/Cargo.lock

```diff
@@ -0,0 +1,20 @@
+[[package]]
+name = "alloc"
+version = "0.0.0"
+dependencies = ["compiler_builtins", "core"]
+
+[[package]]
+name = "compiler_builtins"
+version = "0.1.32"
```

Comment:
Do we need to take special care with updating this on rust releases?

Reply:
I'm not sure :-/

Reply:
Regardless, I'll add a script for easily updating the Cargo.lock

Reply:
Do you have a plan for this? Writing that script sounds non-trivial to me!

Reply:
I don't have a plan for updating the version numbers in Cargo.toml, but I plan to at least write an update script to generate the Cargo.lock from the Python script (using Cargo)

Speaking of which, is there some place I should put my name so I'm notified when there are `build-support/rust` PRs in Nixpkgs? That would make it easier for me to maintain the sysroot code :-)

Reply:
> Speaking of which, is there some place I should put my name so I'm notified when there are build-support/rust PRs in Nixpkgs?

Ah, looks like that place is codeowners, but GitHub requires excessive permissions for that to be useful. I plan to follow nixpkgs-unstable while locally working on Redox, so I should catch some sysroot regressions when they happen. More sysroot testing in the future could also help prevent issues.